lib/ansible/modules/storage/netapp/na_ontap_interface.py

```diff
@@ -156,26 +156,27 @@ class NetAppOntapInterface(object):
     def __init__(self):
 
         self.argument_spec = netapp_utils.na_ontap_host_argument_spec()
-        self.argument_spec.update(
-            state=dict(type='str', default='present', choices=['absent', 'present']),
-            interface_name=dict(type='str', required=True),
-            home_node=dict(type='str'),
-            home_port=dict(type='str'),
-            role=dict(type='str'),
-            address=dict(type='str'),
-            netmask=dict(type='str'),
-            vserver=dict(type='str', required=True),
-            firewall_policy=dict(type='str'),
-            failover_policy=dict(type='str'),
-            admin_status=dict(type='str', choices=['up', 'down']),
-            subnet_name=dict(type='str'),
-            is_auto_revert=dict(type='bool'),
-            protocols=dict(type='list'),
-        )
+        self.argument_spec.update(dict(
+            state=dict(required=False, choices=[
+                       'present', 'absent'], default='present'),
+            interface_name=dict(required=True, type='str'),
+            home_node=dict(required=False, type='str', default=None),
+            home_port=dict(required=False, type='str'),
+            role=dict(required=False, type='str'),
+            address=dict(required=False, type='str'),
+            netmask=dict(required=False, type='str'),
+            vserver=dict(required=True, type='str'),
+            firewall_policy=dict(required=False, type='str', default=None),
+            failover_policy=dict(required=False, type='str', default=None),
+            admin_status=dict(required=False, choices=['up', 'down']),
+            subnet_name=dict(required=False, type='str'),
+            is_auto_revert=dict(required=False, type=bool, default=None),
```

Comment:
should be 'bool'

Reply:
@carchi8py @lonico can we get this committed so we can get this in?

Reply:
I don't have write access to the repo or I would have opened a PR ;)

Reply:
```suggestion
            is_auto_revert=dict(required=False, type='bool', default=None),
```